style.css

```diff
@@ -1225,6 +1334,123 @@
     clear: both;
   }
 
+  .project-modal {
```

Comment:
- [ ] I would like to request you to please stick with the instructions according to which add the new features but your previous work should not get affected or your design should remain the same and looks similar to provided `Figma` design. But it seems like you are not following the design and your design looks bad, it is very important to pay attention to the project details in order to get fruitful results. Kindly have a look at the below table according to which your project card image should be at the alternate position in your desktop project cards. 

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-30 21-42-02](https://github.com/NoelLincoln/mportfolio/assets/66476329/af92e302-ce67-4fc2-ad08-7e36dcf39359)

</td>
<td>

![Screenshot from 2023-06-30 21-41-26](https://github.com/NoelLincoln/mportfolio/assets/66476329/f93eb2d3-8472-47cf-9d8f-239cbb358f34)

</td>
</tr>
</table>